alignak/objects/businessimpactmodulation.py

```diff
@@ -70,13 +70,22 @@ class Businessimpactmodulation(Item):
                        'modulation_period':               StringProp(default=''),
                        })
 
+    def __init__(self, params=None, parsing=True):
+        super(Businessimpactmodulation, self).__init__(params, parsing=parsing)
+
+        # Ok just put None as modulation_period, means 24x7
+        if not hasattr(self, 'modulation_period'):
+            self.modulation_period = '24x7'
```

Comment:
I will have a look to this but it seems that BI modulation have some specific behavior ...